api/apiFunctions.interface.ts

```diff
@@ -41,8 +41,11 @@ export interface IWeeklyPicks {
   gameWeekId: string;
   userResults: IUserPicksData;
 }
+// Currently, the pick history section on the
+// Picks page uses this interface without requiring the teamId. Future
+// accessibility fixes will most likely make use of the teamId.
 export interface INFLTeam {
```

Comment:
What else uses `INFLTeam`? Making this optional will cause Type issues down the long wrong. I'd suggest you keep this intact, create a new interface that extends INFLTeam and makes `teamId` required for the code that is using teamId as a requirement.